docs/product-user-guide/src/main/asciidoc/guided-rules-create-proc.adoc

```diff
@@ -1,25 +1,33 @@
-[id='guided_rules_create_proc']
-= Creating Guided Rules
+[id='guided-rules-create-proc']
+= Creating guided rules
 
 Guided rules enable you to define business rules in a structured format, based on the data objects associated with the rules. You can create and define guided rules individually for your project.
 
 .Procedure
 . Go to *Menu* -> *Design* -> *Projects* and click the project name.
 . Click *Create New Asset* -> *Guided Rule*.
-. Enter an informative *Guided Rule* name and select the appropriate *Package*. You can check *Use Domain Specific Language (DSL)* to use DSL. For details, see <<rules_dsl_con>>.
+. Enter an informative *Guided Rule* name and select the appropriate *Package*. The package that you specify must be the same package where the required data objects have been assigned or will be assigned.
 +
-The new Guided Rule is now created under the selected project.
-
-. Click the *Data Objects* tab and confirm that all data objects required for your rules are listed. If not, click *New item* to import objects from other packages, or **create new data objects(expand content from data-objects-create-proc.adoc) within the package.
-. After all data objects are in place, return to the *Editor* tab and use the buttons on the right of the editor window to add and define the *WHEN* (condition) and *THEN* (action) sections of the rule, based on the available data objects.
+You can also select *Use Domain Specific Language (DSL)* if any DSL assets have been defined in your project (in the *Domain Specific Language Definitions* panel in the *Project Explorer*). These DSL assets will then become usable objects for conditions and actions that you define in the guided rule designer.
```

Comment:
Mention of "Project Explorer" again.. just commenting re: my earlier comment about explicit mention of it; vs "Library".